### PR TITLE
XML removal (part 5)

### PR DESCRIPTION
**Pull request by fgalan**
Part of #1862 

@kzangeli 
@crbrox

### CODE REVIEW

File: test/functionalTest/cases/0000_content_related_headers/accept_fail.test [hunk 1]
```diff
@@ -0,0 +1,305 @@
+# Copyright 2013 Telefonica Investigacion y Desarrollo, S.A.U
```
Comment (fgalan) [inline test/functionalTest/cases/0000_content_related_headers/accept_fail.test]: Not really a new file, but a wrongly detected modification + mv.

NTC (just informative)

File: test/functionalTest/cases/0000_content_related_headers/in_out_formats.test [hunk 2]
```diff
@@ -0,0 +1,316 @@
+# Copyright 2014 Telefonica Investigacion y Desarrollo, S.A.U
```
Comment (fgalan) [inline test/functionalTest/cases/0000_content_related_headers/in_out_formats.test]: Not really a new file, but a wrongly detected modification + mv.

NTC (just informative)

File: test/functionalTest/cases/0000_content_related_headers/missing_content_type_header.test [hunk 3]
```diff
@@ -0,0 +1,111 @@
+# Copyright 2013 Telefonica Investigacion y Desarrollo, S.A.U
```
Comment (fgalan) [inline test/functionalTest/cases/0000_content_related_headers/missing_content_type_header.test]: Not really a new file, but a wrongly detected modification + mv.

NTC (just informative)

**Comment by kzangeli** [comment]
LGTM

**Comment by crbrox** [comment]
LGTM

**Pull request merged**